### DISCUSSION TITLE
Add Important option on tailwind forms plugin

### DISCUSSION THREAD

**Discussion by Luigidefra** [tailwindlabs/tailwindcss#12355] [Ideas]
i need to use tailwind only in a part of my page

I used tailwind important option with class  strategy

```
module.exports = {
  .....
    important: '.tw',
    .....
    }
```

Then all html tags inside tw class parent are managed from tailwind

I need disable preflight option on entire page and insert it too inside tw class parent

so:

```
module.exports = {
    corePlugins: {
        preflight: false,
    },
    important: '.tw',
    }
```
    
 and then on app.scss
  ```
  @tailwind base;
    .tw{
       preflight rules
    }
    @tailwind components;
    @tailwind utilities;
```

on tailwind.config.js i set tailwind forms plugin with class strategy

```
   plugins: [
        require('@tailwindcss/forms')({ strategy: 'class' })
    ]
```

so priority is:
```
.form-input (tailwind-form)
.tw input (preflight)
```

instead it should be
```
.tw input (preflight)
.tw .form-input (tailwind-form)
```

or without class strategy:
```
.tw input (preflight)
.tw input (tailwind-form)
```

Sorry for my english, I hope you understand

**Comment by jnarowski** [comment]
Did you ever figure out a good solution to this? I need it too.
